packages/editor-ui/src/stores/posthog.store.ts

```diff
@@ -161,10 +161,17 @@ export const usePostHog = defineStore('posthog', () => {
 		trackedDemoExp.value[name] = variant;
 	};
 
+	const capture = (event: string, properties: IDataObject) => {
```

Comment:
if we are going to remove the PH hooks file, we also need to make sure we add all the functionality there...
identify https://github.com/n8n-io/n8n-hosted-backend/pull/565/files#diff-3e7bba39ebab6689ae6e79d5920af1871e8fc90da4e0698af49355fc521aaf5dL18
setMetadata https://github.com/n8n-io/n8n-hosted-backend/pull/565/files#diff-3e7bba39ebab6689ae6e79d5920af1871e8fc90da4e0698af49355fc521aaf5dL59
and make sure credentials are still not captured by variables

Reply:
Let's sync about this in our 1:1